src/main/java/uk/gov/hmcts/darts/event/service/impl/HandlerNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.darts.event.service.impl;
```

Comment:
Should this be moved under `package uk.gov.hmcts.darts.event.exception`?

Reply:
yes and also should be using Chris B new exceptions. will update.